src/components/ui/Tiles/Tiles.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { forwardRef } from 'react'
+import type { HTMLAttributes } from 'react'
+
+import Tile from './Tile'
+
+export interface TilesProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+}
+
+const MIN_CHILDREN = 2
+const MAX_CHILDREN = 4
+const NUMBER_ITEMS_TO_EXPAND = 3
+
+const Tiles = forwardRef<HTMLDivElement, TilesProps>(function Tiles(
+  { testId = 'store-tiles', children, ...otherProps },
+  ref
+) {
+  const childrenCount = React.Children.count(children)
+  const spanFirstChild = childrenCount === NUMBER_ITEMS_TO_EXPAND
+  const isOutOfBounds =
+    childrenCount < MIN_CHILDREN || childrenCount > MAX_CHILDREN
+
+  if (isOutOfBounds) {
+    throw new Error(
+      `Tiles cannot receive less than ${MIN_CHILDREN} or more than ${MAX_CHILDREN} children.`
+    )
+  }
+
+  React.Children.forEach(children, (child) => {
+    if (!React.isValidElement(child) || child.type !== Tile) {
+      throw new Error('Only Tile components allowed as children.')
+    }
+  })
+
+  return (
+    <div ref={ref} data-store-tiles data-testid={testId} {...otherProps}>
+      {React.Children.map(children, (child, index) => {
+        if (!React.isValidElement(child)) return
```

Comment:
This is already being checked at the conditional above.

Reply:
Agree with you. 
Just kept this line because `eslint` complains if we didn't check it again. In this case, I prefer to keep than add some `eslint` ignore rules.

![UquutCuABM](https://user-images.githubusercontent.com/11325562/146221863-de54bf0d-ec8f-4244-96de-c007a3fed181.gif)